### ISSUE TITLE
CC Montior support as Target for Display Link

### ISSUE THREAD

**Issue by Luke1505** [tweaked-programs/cccbridge#43]

**Comment by stacode123** [comment]
Yes cool idea i no longer would have to create a separate computer for every of my display

**Comment by Plasman888** [comment]
Yess im trying to write a code for this exact purpose right now but i cant get the getLine command working

**Comment by Plasman888** [comment]
Finally did it, https://pastebin.com/7hmKMNXY;

local monitor = peripheral.find("monitor")
local display = peripheral.find("create_target")
while true do
    monitor.clear()
    a = display.getLine(1)
    monitor.setCursorPos(1, 1)
    monitor.write(a)
    sleep(0.05)
end
